src/common/components/entry-tip-btn/index.tsx

```diff
@@ -1,48 +1,51 @@
 import React, { Component } from "react";
-
 import { Modal } from "react-bootstrap";
-
-import { Global } from "../../store/global/types";
-import { DynamicProps } from "../../store/dynamic-props/types";
-import { ActiveUser } from "../../store/active-user/types";
 import { Account } from "../../store/accounts/types";
-import { Entry } from "../../store/entries/types";
-import { User } from "../../store/users/types";
-import { ToggleType, UI } from "../../store/ui/types";
+
 import { Transactions } from "../../store/transactions/types";
 
+import { Entry } from "../../store/entries/types";
+import { match } from "react-router";
```

Comment:
why reorganize imports? The way we import, we keep package imports on top and then types, components, utils, etc. in order, so imports stay clean.